templates/pages/cv.md

```diff
@@ -0,0 +1,22 @@
+# Cyclic Voltammograms
+
+Select a graph for more details.
+
+<!--
```

Comment:
What's the plan with this comment?

I think it should be removed (and turned into an issue.)

Reply:
This is now https://github.com/echemdb/website/issues/160.